backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientMutationResolver.java

```diff
@@ -27,6 +28,7 @@
 /** Mutations for creating and updating patient records. */
 @Component
 public class PatientMutationResolver implements GraphQLMutationResolver {
+  private final String PERSON_DISPLAY_NAME = "Person";
```

Comment:
This makes me think that the "what type of address this is" constants should all be collected someplace, and possibly that someplace should be an enumeration within either `Translators` or `AddressValidationService`.